contracts/03-tamagotchi-nft/tests/owning.rs

```diff
@@ -1,12 +1,75 @@
 use gtest::{Program, System};
+use gstd::ActorId;
+use tamagotchi_nft_io::{Tamagotchi, TmgAction};
 
-// TODO: 0️⃣ Copy tests from the previous lesson and push changes to the master branch
+const FILL_PER_FEED: u64 = 1000;
+const FILL_PER_ENTERTAINMENT: u64 = 1000;
+const FILL_PER_SLEEP: u64 = 1000;
+
+const NEW_OWNER: ActorId = ActorId::new([0x01; 32]);
+const APPROVED_ACCOUNT: ActorId = ActorId::new([0x02; 32]);
 
 #[test]
 fn owning_test() {
```

Comment:
tests should also be written for bad cases:
a non-owner account is trying to `Transfer` Tamagotchi, `Approve` or `RevokeApproval`

Reply:
New tests added